embulk-output-bigquery.gemspec

```diff
@@ -15,12 +15,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # the latest version
-  spec.add_dependency 'google-api-client','= 0.53.0'
   spec.add_dependency 'time_with_zone'
   spec.add_dependency 'thwait'
-  # activesupport require Ruby >= 2.7.0
+  # activesupport, google-apis-storage_v1, and google-apis-bigquery_v2 require Ruby >= 2.7.0
   # jruby-9.3.0.0 is MRI 2.6 compatible
   spec.add_dependency 'activesupport', "< 7.0"
+  spec.add_dependency 'google-apis-storage_v1','< 0.34.0'
+  spec.add_dependency 'google-apis-bigquery_v2','< 0.65.0'
 
```

Comment:
What do you think about removing the version? Do you have any concern?

* JRuby 9.3.X: will install `google-apis-storage_v1 0.32.0` and `google-apis-bigquery_v2 0.61.0`
* JRuby 9.4.X: will install the latest (Ruby 3.1 compatible)

```
  spec.add_dependency 'google-apis-storage_v1'
  spec.add_dependency 'google-apis-bigquery_v2'
```

Reply:
@hiroyuki-sato 
Thanks for the quick check. That looks good!
I fixed it that way.
272d582